fuzzamoto-ir/src/operation.rs

```diff
@@ -65,6 +78,10 @@ pub enum Operation {
     BuildOpReturnScripts,
     BuildPayToAnchor,
 
+    // filter building operations
+    BuildFilterAddFromTx,
+    BuildFilterAddFromTxo,
```

Comment:
I think to make this complete, we should add some operations for building `FilterLoad` as well. Something like: `BeginBuildFilterLoad`, `AddTxToFilter`, `AddTxoToFilter`, `EndBuildFilterLoad`, similar to the tx building ops.

Reply:
Added 👍